examples/flask/server.py

```diff
@@ -1,36 +1,33 @@
-from argparse import ArgumentParser
+import json
+from datetime import datetime
 from random import randrange
 from time import sleep
 
-from flask import Flask, render_template
+import requests
+from flask import Flask, render_template, jsonify
 from flask_sqlalchemy import SQLAlchemy
 from pybrake.middleware.flask import init_app
 
-parser = ArgumentParser()
-parser.add_argument(
-    "-project_id",
-    dest="project_id",
-    help="airbrake project ID"
-)
-parser.add_argument(
-    "-project_key",
-    dest="project_key",
-    help="airbrake project key"
-)
-args = parser.parse_args()
-
 app = Flask(__name__)
 
 app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:////tmp/test.db"
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 db = SQLAlchemy(app)
 
 app.config["PYBRAKE"] = dict(
-    project_id=args.project_id, project_key=args.project_key,
-    environment="test"
+    project_id="XXXXXXXXXXXX",
+    project_key="XXXXXXXXXXXX",
+    environment="test",
+    error_notifications=True,  # False to disable error notification
```

Comment:
I don't think we want to advertise these options. Disabling errors is almost never what the user wants. Performance stats may not be applicable to some people, so it makes sense to leave them. I would remove lines 23, and 24, but keep line 22.

Reply:
Even `query_stats` and `queue_stats` cannot be disabled by the user?

Added those option over here based on https://github.com/airbrake/pybrake/blob/27c8906e8fe7e6e36faac3c267dbbf0f8184a15e/src/pybrake/notifier.py#L49

https://github.com/airbrake/pybrake/blob/27c8906e8fe7e6e36faac3c267dbbf0f8184a15e/src/pybrake/notifier.py#L50

Reply:
They can be disabled but for the simplicity of the demo I would remove it. We actually don't want to encourage them to disable those options, since that doesn't help with overall application monitoring.

Reply:
Got it. I'll update.

Reply:
Got it. I'll update.